src/main/java/io/github/ImpactDevelopment/installer/Args.java

```diff
@@ -0,0 +1,26 @@
+package io.github.ImpactDevelopment.installer;
+
+import com.beust.jcommander.Parameter;
+import io.github.ImpactDevelopment.installer.setting.InstallationConfig;
+
+public class Args {
+
+    @Parameter(names = { "--no-gpg", "--disable-gpg" }, description = "Disable checking the release signature for testing purposes")
+    public boolean gpg = true;
+
+    @Parameter(names = { "-i", "--impact-version" }, description = "The Impact version to default to")
+    public String impactVersion;
+
+    @Parameter(names = { "-m", "--minecraft-version" }, description = "The Minecraft version to default to")
+    public String minecraftVersion;
```

Comment:
I suppose yeah, could call it like, uh, `--dryrun` where it goes through the motions and makes sure everything works (really just testing the refmap and the hashes to make sure they're right).

Depending on how we fix the two things I mentioned here https://github.com/ImpactDevelopment/Impact/pull/6#issuecomment-489790797 this could be useful or redundant -- maybe we just have the metadata outputter verify the hashes itself.